web/modules/custom/joinup_user/joinup_user.module

```diff
@@ -342,8 +488,24 @@ function joinup_user_og_membership_presave(EntityInterface $entity) {
 }
 
 /**
- * Implements hook_ENTITY_TYPE_predelete() for OG Membership entities.
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Disallow access to the cancellation method.
  */
-function joinup_user_og_membership_predelete(EntityInterface $entity) {
-  joinup_user_og_membership_presave($entity);
+function joinup_user_form_user_multiple_cancel_confirm_alter(&$form, FormStateInterface $form_state, $form_id) {
+  joinup_user_form_user_cancel_form_alter($form, $form_state, $form_id);
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Disallow access to the cancellation method.
+ */
+function joinup_user_form_user_cancel_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
Ohh I get it now, thanks for explaining. I confused this with the account settings admin form which also shows this option.